src/packages/lib.ts

```diff
@@ -20,8 +20,8 @@ export { default as GridToggler } from '../h5web/toolbar/controls/GridToggler';
 export { default as VisCanvas } from '../h5web/vis-packs/core/shared/VisCanvas';
 export { default as PanZoomMesh } from '../h5web/vis-packs/core/shared/PanZoomMesh';
 export { default as TooltipMesh } from '../h5web/vis-packs/core/shared/TooltipMesh';
+export { default as VisMesh } from '../h5web/vis-packs/core/shared/Mesh';
```

Comment:
I moved `Mesh` in `shared` so that it does not make much sense to export it as `HeatmapMesh`. `VisMesh` sounded appropriate to me...